ros2topic/ros2topic/api/__init__.py

```diff
@@ -37,6 +38,26 @@ def unsigned_int(string):
     return value
 
 
+def nonnegative_int(inval):
+    ret = int(inval)
+    if ret < 0:
+        # The error message here gets completely swallowed by argparse
```

Comment:
maybe, use `ArgumentTypeError` instead

Reply:
according to argparse docs, a raised `ValueError` should be formatted correctly, but if that's not the case let's use `ArgumentTypeError`.